app/views/conservation_records/treatment_report_pdf.html.erb

```diff
@@ -18,91 +21,118 @@ table {
+      </tr>
+      <tr><td><br></td></tr>
+      <tr>
+	<th>Author</th>
+      </tr>
+      <tr>
+	<td><%= @conservation_record.author %></td>
       </tr>
     </table>
   </div>
   <br>
-  <h2>Description</h2>
-  <strong>General Remarks</strong> <%= simple_format(@conservation_record.treatment_report.description_general_remarks) %>
-  <strong>Binding</strong> <%= simple_format(@conservation_record.treatment_report.description_binding) %>
-  <strong>Textblock</strong> <%= simple_format(@conservation_record.treatment_report.description_textblock) %>
-  <strong>Primary Support</strong> <%= simple_format(@conservation_record.treatment_report.description_primary_support) %>
-  <strong>Medium</strong> <%= simple_format(@conservation_record.treatment_report.description_medium) %>
-  <strong>Attachments | Inserts</strong> <%= simple_format(@conservation_record.treatment_report.description_attachments_inserts) %>
-  <strong>Housing</strong> <%= simple_format(@conservation_record.treatment_report.description_housing) %>
-
-  <h2>Condition</h2>
-  <strong>Summary</strong> <%= simple_format(@conservation_record.treatment_report.condition_summary) %>
-  <strong>Binding</strong> <%= simple_format(@conservation_record.treatment_report.condition_binding) %>
-  <strong>Textblock</strong> <%= simple_format(@conservation_record.treatment_report.condition_textblock) %>
-  <strong>Primary Support</strong> <%= simple_format(@conservation_record.treatment_report.condition_primary_support) %>
-  <strong>Medium</strong> <%= simple_format(@conservation_record.treatment_report.condition_medium) %>
-  <strong>Housing</strong> <p><%= friendly_housing(@conservation_record.treatment_report.condition_housing_id) %></p>
-  <strong>Housing Narrative</strong> <%= simple_format(@conservation_record.treatment_report.condition_housing_narrative) %>
-  <strong>Attachments | Inserts</strong> <%= simple_format(@conservation_record.treatment_report.condition_attachments_inserts) %>
-  <strong>Previous Treatment</strong> <%= simple_format(@conservation_record.treatment_report.condition_previous_treatment) %>
-  <strong>Materials Analysis</strong> <%= simple_format(@conservation_record.treatment_report.condition_materials_analysis) %>
-
-  <h2>Treatment Proposal</h2>
-  <strong>Proposal</strong> <%= simple_format(@conservation_record.treatment_report.treatment_proposal_proposal) %>
-  <strong>Housing Need</strong> <p><%= friendly_housing(@conservation_record.treatment_report.treatment_proposal_housing_need_id) %></p>
-  <strong>Factors Influencing Treatment</strong> <%= simple_format(@conservation_record.treatment_report.treatment_proposal_factors_influencing_treatment) %>
-  <strong>Performed Treatment</strong> <%= simple_format(@conservation_record.treatment_report.treatment_proposal_performed_treatment) %>
-  <strong>Housing Provided</strong> <p><%= friendly_housing(@conservation_record.treatment_report.treatment_proposal_housing_provided_id) %></p>
```

Comment:
Same as comment above.  Add `.key` to the end.

Reply:
Done.